server/src/main/java/io/druid/server/metrics/MetricsModule.java

```diff
@@ -95,4 +101,40 @@ public MonitorScheduler getMonitorScheduler(
         monitors
     );
   }
+
+  @Provides
+  @ManageLifecycle
+  public JvmMonitor getJvmMonitor(Properties props)
+  {
+    return new JvmMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public JvmCpuMonitor getJvmCpuMonitor(Properties props)
+  {
+    return new JvmCpuMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public SysMonitor getSysMonitor(Properties props)
+  {
+    return new SysMonitor(getDimensions(props));
+  }
+
+  private Map<String, String[]> getDimensions(Properties props)
+  {
+    Map<String, String[]> dimensions = new HashMap<>();
+    for (String property : props.stringPropertyNames()) {
+      if (property.startsWith(MonitorsConfig.METRIC_DIMENSION_PREFIX)) {
+        dimensions.put(
+            property.substring(MonitorsConfig.METRIC_DIMENSION_PREFIX.length()),
+            new String[]{props.getProperty(property)}
```

Comment:
I verified it with loggingEmitter, 
the key we got is from an existing property, afaik in this case the value cannot be null here, am i missing anything ?